### ISSUE TITLE
Training Data

### ISSUE THREAD

**Issue by ChrisBobotsis** [ChrisBobotsis/Mario-Kart-Neural-Network#3]
Training Data as is is largely unbalanced. Of course I can just make the ratios of all possible classifiers the same, but there will be virtually no 'no input' inputs. So I'm not quite sure how to deal with that.

1. I could add in random 'no input' and see if that helps.
2. I can also mirror image training data and thus invert the key board presses to double the dataset. For example an image+[left,fwd] would be inv(image)+[right,fwd].
3. Make sure to also include images of Lakitu showing the lap you are on so the dataset is used to him being on screen.

**Comment by ChrisBobotsis** [comment]
After the first epoch accuracy increases for the next 9 epochs about 10%. This is with ~70k training samples. Therefore I don't think adding more training data will help unless it is from different tracks.
